sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -180,6 +180,10 @@ public void setTransaction(final @Nullable String transaction) {
     set("sentry-transaction", transaction);
   }
 
+  public void setSampleRate(final @Nullable String sampleRate) {
+    set("sentry-samplerate", sampleRate);
```

Comment:
We decided to make baggage and envelope header key names identical (except for the `sentry-` prefix in baggage keys) so that we're able to just dump the DSC data into both transport mechanisms without any mapping. (Linking to the preview here for [more details](https://develop-git-lforst-dynamic-sampling-context.sentry.dev/sdk/performance/dynamic-sampling-context/#baggage-header)) 

Therefore, we should change the snake case keys to underscores (like in the example below)
```suggestion
    set("sentry-sample_rate", sampleRate);
```

Reply:
Yes, exactly

Reply:
> * `sentry-traceid` becomes `sentry_trace_id`

It becomes `sentry-trace_id`, but that probably just was a typo ^^

Reply:
OK thanks, will updated.
Yeah just a typo on the `_` there.